public/mixed_washu_nejm_2016/meta_study.txt

```diff
@@ -0,0 +1,5 @@
+type_of_cancer: mixed
```

Comment:
should it be `mnm` (Myeloid Neoplasm) since it is parent of both `aml` and `mds`?